packages/store/src/space.ts

```diff
@@ -57,6 +57,8 @@ export class Space {
     rootDeleted: new Signal<string>(),
     textUpdated: new Signal<Y.YTextEvent>(),
     updated: new Signal(),
+    // eslint-disable-next-line @typescript-eslint/no-explicit-any
+    attribute: new Signal<Map<string, any>>(),
```

Comment:
* `attribute` -> `attributes`
* Can we replace `any` with `unknown`?

Reply:
cool, I will repair it

Reply:
I have done the rename stuff, just checkout the master branch :)